EnglishTranslation-master/ITEM.tsv

```diff
@@ -9601,111 +9601,111 @@ ITEM_20151224_009600	Spell Detector Orb
+ITEM_20151224_009638	A delivery request for ointment, sent from Settler Brophen to Settler Layla.
+ITEM_20151224_009639	Leaves with a slight reddish hue. Settler Izna said that these are great for making an ointment for wounds.
 ITEM_20151224_009640	Mucus with an odd smell. An ingredient used to create a fatigue recovery potion. Really...?
 ITEM_20151224_009641	Useful Objects
-ITEM_20151224_009642	Objects left by lumberjacks during their retreat. Some of them might be useful if they're fixed correctly. 
+ITEM_20151224_009642	Objects left by fleeing lumberjacks. Some of them might be useful if they're repaired. 
 ITEM_20151224_009643	Priest Irma's Journal Page
-ITEM_20151224_009644	A page from Priest Irma's journal that has been bitten and torn by monsters.{nl}Is it even readable at all?
+ITEM_20151224_009644	A page from Priest Irma's journal that's been bitten and torn by monsters.{nl}Is it even readable at all?
 ITEM_20151224_009645	Priest Irma's Journal
 ITEM_20151224_009646	Collected pages of Priest Irma's Journal.
-ITEM_20151224_009647	A note found in a bundle of grass with smoke coming out. Half of the content is burnt. 
-ITEM_20151224_009648	A drug created coincidentally from a failure by Germeja. The monsters will fight against each other if used on them.
+ITEM_20151224_009647	A note found in a smoking bundle of grass. Half of the content is burnt. 
+ITEM_20151224_009648	A drug created coincidentally from Germeja's failure. The monsters will fight against each other if it's used on them.
 ITEM_20151224_009649	A note left by Priest Gelija. This might be the decisive clue since Gelija is so thorough.
 ITEM_20151224_009650	A whistle that can call Jolly to your location. Blow the whistle if Jolly is too far away.
 ITEM_20151224_009651	Leftover Box
-ITEM_20151224_009652	The blood of monsters found in Rohonxa Cliff. Is it contaminated due to the demons?
+ITEM_20151224_009652	The blood of monsters found in Rohonsa Cliff. Is it contaminated due to the demons?
 ITEM_20151224_009653	Chaser's Package
 ITEM_20151224_009654	A bag filled with medicine and bandages.
-ITEM_20151224_009655	A red crystal once held by demons. Once could very well be threatened because of it.
-ITEM_20151224_009656	A wood piece with a corruptive energy held by the demons. Can it drive out the curse...
-ITEM_20151224_009657	A mysterious pack held by every demons. It might be related to the curse.
-ITEM_20151224_009658	A mysterious pack held by every demons. Try using it figure out the effect.
+ITEM_20151224_009655	A red crystal once held by demons. It wouldn't be out of the question to feel threatened by it.
+ITEM_20151224_009656	A wood piece held by demons emitting a corruptive energy. Can it drive out the curse?
+ITEM_20151224_009657	A mysterious pack held by every demon. It might be related to the curse.
+ITEM_20151224_009658	A mysterious pack held by every demon. Try using it figure what it does. Don't try this at home!
```

Comment:
Try using it figure what it does -> Try using it to figure out what it does.
"Don't try this at home!" is contradicting. The description says to open it, but then followed by a sentence not to.